src/SystemCommands-RefactoringSupport/RBRefactoringError.extension.st

```diff
@@ -1,5 +1,16 @@
 Extension { #name : 'RBRefactoringError' }
 
+{ #category : '*SystemCommands-RefactoringSupport' }
+RBRefactoringError >> failedConditions [
+	^ failedConditions
+]
+
+{ #category : '*SystemCommands-RefactoringSupport' }
+RBRefactoringError >> faledConditions: aCollection [
+	"aCollection should contain objects of kind RBNewAbstractCondition"
+	failedConditions := aCollection
+]
+
```

Comment:
Why are these defined as extension methods?  I checked and it works fine if they are not extensions. Would it make sense to put them in `RBRefactoringError`?

Reply:
This was a mistake, thanks for pointing this out. I just fixed it